cypress/fixtures/checkAnswersRows.json

```diff
@@ -39,6 +39,10 @@
     {
       "key": "Registered to vote",
       "value": "No"
+    },
+    {
+      "key": "Do you confirm that your income was less than $12,070?",
+      "value": "Yes"
```

Comment:
This one is fine, but we also want to move the "small and rural community" key above the other trillium benefits.

Reply:
ah shoot, it's because I lowercased "revenue" and forgot to update the test 🤦‍♀️